CHANGELOG.md

```diff
@@ -13,6 +13,30 @@ next release
 
 ### UI Changes
 
+1.33.0 (2022-04-11)
+-------------------
+### Backend Changes
+
+* Elasticsearch: Only create ILM template if configured. ([@rbizos](https://github.com/rbizos) in [#3610](https://github.com/jaegertracing/jaeger/pull/3610))
```

Comment:
should this be under Bugs?

Reply:
ok, yeah, i was unsure at first but i've read the PR more now and i agree.